docs/devel/releasing.md

```diff
@@ -1,307 +0,0 @@
-- beta release: `export RELEASE_VERSION="vX.Y.Z-beta.W"`;
-- official release: `export RELEASE_VERSION="vX.Y.Z"`;
-- new release series: `export RELEASE_VERSION="vX.Y"`.
-
-Then, run
-
-```console
-./release/cut-official-release.sh "${RELEASE_VERSION}" "${BUILD_VERSION}"
-```
-
-This will do a dry run of the release.  It will give you instructions at the
-end for `pushd`ing into the dry-run directory and having a look around.
-`pushd` into the directory and make sure everythig looks as you expect:
-
-```console
-git log "${RELEASE_VERSION}"  # do you see the commit you expect?
-make release
-./cluster/kubectl.sh version -c
-```
-
-If you're satisfied with the result of the script, go back to `upstream/master`
-run
-
-```console
-./release/cut-official-release.sh "${RELEASE_VERSION}" "${BUILD_VERSION}" --no-dry-run
-```
-
-and follow the instructions.
-
-### Publishing binaries and release notes
-
-Only publish a beta release if it's a standalone pre-release (*not*
-vX.Y.Z-beta.0).  We create beta tags after we do official releases to
-maintain proper semantic versioning, but we don't publish these beta releases.
-
-The script you ran above will prompt you to take any remaining steps to push
-tars, and will also give you a template for the release notes.  Compose an
-email to the team with the template.  Figure out what the PR numbers for this
-release and last release are, and get an api-token from GitHub
-(https://github.com/settings/tokens).  From a clone of
-[kubernetes/contrib](https://github.com/kubernetes/contrib),
-
-```
-go run release-notes/release-notes.go --last-release-pr=<number> --current-release-pr=<number> --api-token=<token> --base=<release-branch>
-```
-
-where `<release-branch>` is `master` for alpha releases and `release-X.Y` for beta and official releases.
-
-**If this is a first official release (vX.Y.0)**, look through the release
-notes for all of the alpha releases since the last cycle, and include anything
-important in release notes.
-
-Feel free to edit the notes, (e.g. cherry picks should generally just have the
-same title as the original PR).
-
-Send the email out, letting people know these are the draft release notes.  If
-they want to change anything, they should update the appropriate PRs with the
-`release-note` label.
-
-When you're ready to announce the release, [create a GitHub
-release](https://github.com/kubernetes/kubernetes/releases/new):
-
-1. pick the appropriate tag;
-1. check "This is a pre-release" if it's an alpha or beta release;
-1. fill in the release title from the draft;
-1. re-run the appropriate release notes tool(s) to pick up any changes people
-   have made;
-1. find the appropriate `kubernetes.tar.gz` in [GCS bucket](https://console.developers.google.com/storage/browser/kubernetes-release/release/),
-   download it, double check the hash (compare to what you had in the release
-   notes draft), and attach it to the release; and
-1. publish!
-
-### Manual tasks for new release series
```

Comment:
AFAIK, you haven't replaced any of this stuff in your new tooling, so it should probably stay/get moved over.